### PR TITLE
Update all exporters to use the new helper TraceExporter.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Only Zipkin is left.

### CODE REVIEW

File: exporter/awsexporter/aws_xray.go [hunk 1]
```diff
@@ -146,25 +155,12 @@ func transformConfigToXRayOptions(axrCfg *awsXRayConfig) (xopts []xray.Option, e
...
-			span.SetStatus(trace.Status{
-				Code:    int32(trace.StatusCodeUnknown),
-				Message: xerr.Error(),
-			})
-		}
-		span.End()
-	}()
-
+func (axe *awsXRayExporter) PushTraceData(ctx context.Context, td data.TraceData) (int, error) {
 	serviceName := td.Node.GetServiceInfo().GetName()
```
Comment (songy23) [inline exporter/awsexporter/aws_xray.go]: Is there a way to refactor the recording of `serviceName` into `PushTraceData`? Maybe add a TODO.

Comment (bogdandrutu) [reply exporter/awsexporter/aws_xray.go]: Do you mean to add it in the exporterhelper decorator?

Comment (songy23) [reply exporter/awsexporter/aws_xray.go]: Yes, like `pushTraceDataWithSpan`.

Comment (bogdandrutu) [reply exporter/awsexporter/aws_xray.go]: DOne.

**Pull request merged**